ansible/roles_ocp_workloads/ocp4_workload_xray_data_pipeline/templates/storage/subscription.yml.j2

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ocs-operator
+  namespace: {{ namespace }}
+spec:
+  channel: {{ ocs_channel }}
+  installPlanApproval: Automatic
+  name: ocs-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
```

Comment:
This should probably have Catalog Snapshot support to lock a version in place.

Reply:
Maybe... Will fix once we know it's working. Right now I'm trying to deploy OCS4.5 as it's GA. If this does not work I'll have to fall back to 4.4.